rawTransaction.go

```diff
@@ -315,3 +334,36 @@ func (txn *MultiAgentWithFeePayerRawTransactionWithData) UnmarshalBCS(des *bcs.D
 
 // endregion
 // endregion
+
+// DeserializeMultiAgentTransaction deserializes a BCS encoded MultiAgent transaction
+func DeserializeMultiAgentTransaction(data []byte) (*RawTransactionWithData, error) {
+	des := bcs.NewDeserializer(data)
+	rawTxnWithData := &RawTransactionWithData{}
+
+	rawTxn := &RawTransaction{}
+	rawTxn.UnmarshalBCS(des)
+
+	if des.Error() != nil {
+		return nil, des.Error()
+	}
+	secondarySigners := bcs.DeserializeSequence[AccountAddress](des)
+
```

Comment:
I see what you mean now.

We should implement Marshal and Unmarshal for RawTransactionWithData probably then.

Is this to deserialize from a signed or unsigned transaction